src/uu/stat/src/stat.rs

```diff
@@ -93,9 +97,30 @@ pub enum OutputType {
     Unknown,
 }
 
+enum QuotingStyle {
+    Locale,
+    Shell,
+    Default,
+    Quote,
+}
```

Comment:
Is it on purpose that not all possible quoting styles (see https://www.gnu.org/software/coreutils/manual/html_node/stat-invocation.html#stat-invocation) are represented as variants?

Reply:
yeah, i will open a follow up bug.
it should also be covered by upstream tests
I think we can probably reuse code from other programs.